### ISSUE TITLE
Album art image in notifications stopped working with Big Sur

### ISSUE THREAD

**Issue by vbrilon** [citruspi/Spotify-Notifications#122]
When I updated to Big Sur, the notifications show all the info but no longer show the album art.
<img width="366" alt="Screen Shot 2020-11-13 at 12 29 45 PM" src="https://user-images.githubusercontent.com/1173618/99118223-fe3d5600-25ab-11eb-97b3-4860538d4ddd.png">

**Comment by widestm** [comment]
Seeing this too. I suspect that this is somehting that spotify needs to fix as Now Playing in the Menu bar and Notification center does not show album art from Spotify either. In https://www.apple.com/macos/big-sur/ you can see that they display some other app that does show album art which could indicate that it's just spotify that is lacking support.

<img width="307" alt="Screenshot 2020-11-18 at 13 37 15" src="https://user-images.githubusercontent.com/6040738/99531821-9b7cfd80-29a3-11eb-8658-f195d0f5bab5.png">

**Comment by jirkap** [comment]
Yes yes, saw that too right away.. Don't need control buttons to skip track so till it's fixed, I'm using older [version 0.5.2](https://github.com/citruspi/Spotify-Notifications/releases/tag/0.5.2) which handles also album cover (just on right side, but I'm OK with it)
![Snímek obrazovky 2020-11-26 v 12 26 10](https://user-images.githubusercontent.com/74545881/100345753-0d2bfb80-2fe3-11eb-83b1-fea67fc3e432.png)
